das/bigcache_storage_service.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package das
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/allegro/bigcache"
+	"github.com/offchainlabs/nitro/arbstate/daprovider"
+	"github.com/offchainlabs/nitro/das/dastree"
+	"github.com/offchainlabs/nitro/util/pretty"
+	flag "github.com/spf13/pflag"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type BigCacheConfig struct {
+	// TODO add other config information like HardMaxCacheSize
+	Enable             bool          `koanf:"enable"`
+	Expiration         time.Duration `koanf:"expiration"`
+	MaxEntriesInWindow int
+}
+
+var DefaultBigCacheConfig = BigCacheConfig{
+	Expiration: time.Hour,
+}
+
+var TestBigCacheConfig = BigCacheConfig{
+	Enable:             true,
+	Expiration:         time.Hour,
+	MaxEntriesInWindow: 1000,
+}
+
+func BigCacheConfigAddOptions(prefix string, f *flag.FlagSet) {
+	f.Bool(prefix+".enable", DefaultBigCacheConfig.Enable, "Enable local in-memory caching of sequencer batch data")
+	f.Duration(prefix+".expiration", DefaultBigCacheConfig.Expiration, "Expiration time for in-memory cached sequencer batches")
+}
+
+type BigCacheStorageService struct {
```

Comment:
This file was actually deleted in master.

Reply:
thanks! my bad, not sure how that got in.